package/src/contexts/messageInputContext/hooks/useMessageDetailsForState.ts

```diff
@@ -44,8 +44,8 @@ export const useMessageDetailsForState = <
   useEffect(() => {
     if (message && !isEditingBoolean<StreamChatGenerics>(message)) {
       setText(message?.text || '');
-      const newFileUploads = [];
-      const newImageUploads = [];
+      const newFileUploads: FileUpload[] = [];
```

Comment:
nice 👍  this is a good practice to follow

Reply:
✅